app/Controller/PagesController.php

```diff
@@ -72,5 +118,59 @@ public function display() {
 			}
 			throw new NotFoundException();
 		}
-	}
+  }
+
+  private $顧客名;
+
+  function set顧客名($v)
+  {
+    $this->顧客名 = $v;
+    return $this;
+  }
+
+  function get顧客名()
+  {
+    return $this->顧客名;
+  }
+}
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
+    public function example()  {
+        if ($a == $b)  {
```

Comment:
__[staging][code_sniffer]__
* Expected "if (...) {\n"; found "if (...)  {\n"
* Operator == prohibited; use === instead
* There must be a single space between the closing parenthesis and the opening brace of a multi-line IF statement; found 2 spaces

<!-- Comment by [SideCI](https://sideci.com) -->